### PR TITLE
CAMEL-14797: Fixes The Quick Look Up issue

### PR DESCRIPTION
**Pull request by psingh587**
- In this PR, I have tried to solve the issue related to Quick Look Up Search Result appearance. An extra arrow used to appear. 
In this I have removed the appearance of arrows when anything is typed in the search box which eradicates the issue of the messy toggle. And the arrows restore to their original places once the text in the search box is NIL.

### CODE REVIEW

File: antora-ui-camel/src/js/05-nav-search.js [hunk 1]
```diff
@@ -6,9 +6,18 @@ document.addEventListener('DOMContentLoaded', function () {
   if (!navSearch) {
     return
   }
-
+  var navToggle = document.querySelectorAll('.nav-item-toggle')
   var navLinks = document.querySelectorAll('.nav-link')
   navSearch.addEventListener('keyup', function () {
+    if (navSearch.value === '') {
+      for (var b = 0; b < navToggle.length; b++) {
```
Comment (zregvart) [inline antora-ui-camel/src/js/05-nav-search.js]:
```suggestion
      for (let i = 0; i < navToggle.length; i++) {
```

Or you can use `https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach`

Could also use single loop with `navSearch.value === '' ? 'block' : 'none'` here.

File: antora-ui-camel/src/js/05-nav-search.js [hunk 2]
```diff
@@ -6,9 +6,18 @@ document.addEventListener('DOMContentLoaded', function () {
...
+        navToggle[b].style.display = 'block'
+      }
+    } else {
+      for (b = 0; b < navToggle.length; b++) {
```
Comment (zregvart) [inline antora-ui-camel/src/js/05-nav-search.js]: Declare variable using `var`/`let`. Could bring `b` into global scope.

**Comment by psingh587** [comment]
@zregvart  Made the changes.

**Comment by zregvart** [comment]
Thanks!

**Pull request merged**